### ISSUE TITLE
FalsePositive for calculated property in DTO

### ISSUE THREAD

**Issue by inyutin-maxim** [inyutin-maxim/System.IO.Abstractions.Analyzers#54]
I think the analyzer should ignore the calculated property in DTO. For Example:

``` cs
public class View
{
	public string Name { get; set; }
	public string Format => Path.GetExtension(Name);
}
```
